### PR TITLE
V2

### PR DESCRIPTION
**Pull request by otherpirate**

### CODE REVIEW

File: bento_map/models.py [hunk 1]
```diff
@@ -124,14 +122,51 @@ def element(self):
...
+    @property
+    def type(self):
+        return "edges"
+
+    @property
+    def element(self):
+        provider = self.environment.replace(TSURU_PRODUCTION_ENVIRONMENT, '')
+        if provider:
+            provider = '-' + provider
+        provider = 'tsuru_tsuru-dbaas' + provider
```
Comment (elesbom) [inline bento_map/models.py]: tsuru_tsuru-dbaas is hardcoded ?

File: bento_map/client.py [hunk 2]
```diff
@@ -13,9 +15,11 @@ def __init__(self):
         self.databases = {}
 
     def _get(self, page=1):
-        url = BENTO_ENDPOINT + '/api/host/'
-        resp = requests.get(url, {'page': page, 'format': 'json'})
-
+        url = BENTO_ENDPOINT + 'api/host/'
```
Comment (elesbom) [inline bento_map/client.py:18]: BENTO_ENDPOINT is endpoint or host ?

**Review by elesbom — APPROVED** [review]

File: bento_map/client.py (hunk 2 above)
Comment (otherpirate) [reply bento_map/client.py:18]: It's endpoint :)

File: bento_map/models.py (hunk 1 above)
Comment (otherpirate) [reply bento_map/models.py]: I will put that in an environment variable, thanks!

**Pull request merged**